backend/src/chat/chat.service.ts

```
@@ -132,3 +154,152 @@
+    private userRepository: Repository<User>,
+    @InjectRepository(Message)
+    private messageRepository: Repository<Message>,
+  ) {}
+
+  async getChatHistory(chatId: string): Promise<Message[]> {
+    const chat = await this.chatRepository.findOne({
+      where: { id: chatId, isDeleted: false },
+      relations: ['messages'],
+    });
+
+    if (chat && chat.messages) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)